server/server/app.py

```diff
@@ -22,6 +22,7 @@ def app_factory() -> Tuple[Api, Flask]:
     api.add_resource(Languages, '/languages')
     api.add_resource(Search, '/search')
     api.add_resource(Menu, '/menu')
+    api.add_resource(SuttaplexList, '/suttaplex-list/<path:uid>')
```

Comment:
Are there going to be other endpoints that start with 'suttaplex'? If not, maybe the '-list' part is redundant?